<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>PasteOptionsBase</class>
 <widget class="QWidget" name="PasteOptionsBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>257</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="label_sep" >
     <property name="text" >
      <string>Sep&amp;arator:</string>
     </property>
     <property name="buddy" >
      <cstring>SeparatorCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="KComboBox" name="SeparatorCombo" >
     <property name="minimumSize" >
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip" >
      <string>Choose which separator you want to use to separate your data.</string>
     </property>
     <property name="whatsThis" >
      <string>Choose one separator which will divide the parts of an expression when transferring data from or to another application over the clipboard.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>310</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QGroupBox" name="label_order" >
     <property name="title" >
      <string>Order</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item rowspan="4" row="1" column="0" >
       <widget class="Q3ListBox" name="OrderList" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>3</hsizetype>
          <vsizetype>3</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QPushButton" name="DownButton" >
        <property name="text" >
         <string>&amp;Down</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QPushButton" name="SkipButton" >
        <property name="text" >
         <string>S&amp;kip</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="UpButton" >
        <property name="text" >
         <string>&amp;Up</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" >
       <widget class="QCheckBox" name="kcfg_UseCurrent" >
        <property name="toolTip" >
         <string>Check this if you want the items in the clipboard to be in the same language order as the current document.</string>
        </property>
        <property name="whatsThis" >
         <string>When checked, the items in the clipboard are assumed to be in the same language order as the current document.</string>
        </property>
        <property name="text" >
         <string>Use &amp;current document</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>21</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>80</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ListBox</class>
   <extends></extends>
   <header>q3listbox.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends></extends>
   <header>kcombobox.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>SeparatorCombo</tabstop>
  <tabstop>kcfg_UseCurrent</tabstop>
  <tabstop>OrderList</tabstop>
  <tabstop>UpButton</tabstop>
  <tabstop>SkipButton</tabstop>
  <tabstop>DownButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
